pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java

```diff
@@ -275,7 +282,9 @@ protected void handleConnected(CommandConnected connected) {
         }
 
         checkArgument(state == State.SentConnectFrame || state == State.Connecting);
-
+        this.maxMessageSize = connected.getMaxMessageSize();
```

Comment:
```
if (connected.hasMaxMessageSize() {
 . ..
} else {
   maxMessageSize = Commands.DEFAULT_MAX_MESSAGE_SIZE;
}

```

Reply:
Thank you for reminder. It' s the optional value in protocol, will fix